_layouts/default.html

```diff
@@ -175,9 +111,7 @@ <h2><span>{{ page.title }}</span></h2>
                                     <!-- Tags widget-->
```

Comment:
This also isn't doing anything?

Reply:
I'm thinking to leave it for now until you change up the sites UI as it gives context for the categories whether we are looking at a list of documents or videos.